### ISSUE TITLE
Implement post-processing

### ISSUE THREAD

**Issue by gcapes** [RoryAtBar/Abaqus_bayesian_matflow#11]
After #10, there is a section in `call_abaqus_with_new_params` which looks to read in results and save them in a dataframe.

These tasks could be done as `output_file_parsers`. See `simulate_VE_loading` here:
https://github.com/hpcflow/matflow-new/blob/eee11b62b37b4b9ad91921ee0f7dc1fb01f2f27a/matflow/data/template_components/task_schemas.yaml#L491

**Comment by RoryAtBar** [comment]
Yes, is this so that we separate reading action of reading the output files from actually running Abaqus? 

That's fine. Do we also want to change storing data from a dataframe to an hdf5? it's a Pandas dataframe because that seemed to be a convenient way of keeping inputs and outputs together, but that's also what the simulate_VE_loading task does

**Comment by gcapes** [comment]
While yes, we do want to separate the running of Abaqus with the post-processing of the output files, it's more just the approach @aplowman suggested instead of creating separate task schemas for these post-processing steps.

**Comment by RoryAtBar** [comment]
So, you're suggesting having a task schema that runs abaqus, and reads the result.

I guess the task you're looking at looking at is this:

`        damask_post_processing_result:
          from_files: [damask_hdf5_file]
          save_files: false
          script: <<script:damask/process_damask_HDF5.py>>
          inputs: [damask_post_processing]`

And you would change the script for one that reads the output files?

Sounds good, there are two aspects to reading the output:

1) running a python script within Abaqus CAE to read the .odb file and generate text files (.rpt).
2) reading the text files into the pandas dataframe

I would have combined this into one step, but abaqus CAE can only use the Abaqus modules and numpy, and the functions that read the .odb files can only be used within Abaqus. Therefore the output files need to be generated by abaqus, then read by abaqus, and then you can use an external python module to create some sort of data base.

I see you have self-assigned this, but is there anything you would like me to take care of to make this easier?

**Comment by gcapes** [comment]
Hi Rory, 
You have probably already seen that I've done #10, so I'm now looking at this, which means just 2. above is outstanding, and it's this bit that I think would be done using `output_file_parsers`.

As for pandas dataframe vs hdf5 ... I don't really feel in a position to advise. It wouldn't be a huge deal to switch from one to the other further down the line. What I'm focussing on initially is to get each step into Matflow, then worry about iterating over the various property values afterwards.

It would probably be good to have you try the workflow soon but I'll get in touch separately about that.
Cheers!
